README.md

```diff
@@ -843,13 +843,45 @@ Params:
 - `steps`: a list of steps/actions to execute in the scope of the for_each loop.
   It's analogous to the `steps` field at the top level of the spec file.
 
+### Ignore (Optional)
+
+We use [filepath Glob](https://pkg.go.dev/path/filepath#Glob) to match the file
+and directory paths that should be ignored if included/copied from template
+directory to destination directory, and should not be overwritten if included
+from destination. This `ignore` feature is similiar to `skip` in `include`
+action, the difference here is that ignore is global and it applies to every
+`include` action.
+
+This section is optional, if not provided, a default ignore list is used:
+`.DS_Store`, `.bin`, and `.ssh`. To set your custom ignore list, please check
+accepted patterns [here](https://pkg.go.dev/path/filepath#Match).
+
+Example:
+
+```yaml
+ignore:
+  - '*/*.ssh'
+steps:
+  - desc: 'Include some files and directories'
+    action: 'include'
+    params:
+      # ignore script files in `src_dir` but not its sub-directories.
+      paths: ['src_dir']
+  - desc: 'Include some files and directories from destination'
+    action: 'include'
+    params:
+      # skip overwriting script files in `dest_dir` but not its sub-directories.
```

Comment:
Ditto here, I think "skip overwriting" might be confusing. Just "don't include" should work.